src/scss/components/ExternalPage.scss

```diff
@@ -41,7 +41,7 @@
 }
 
 .BlockHero--pricing {
-  background-image: url(/img/block-title-text--hero@2x.png);
+  background-image: url(../img/block-title-text--hero@2x.png);
```

Comment:
This change will allow the background image to appear via http://bufferapp.github.io/buffer-style/